tests/test_nan.py

```diff
@@ -12,36 +11,41 @@ class NanTest(dj.Manual):
     """
 
 
-@pytest.fixture(scope="module")
-def schema(connection_test):
-    schema = dj.Schema(PREFIX + "_nantest", connection=connection_test)
+@pytest.fixture
+def schema_nan(connection_test, prefix):
+    schema = dj.Schema(
+        prefix + "_nantest", context=dict(NanTest=NanTest), connection=connection_test
+    )
     schema(NanTest)
     yield schema
     schema.drop()
 
 
-@pytest.fixture(scope="class")
-def setup_class(request, schema):
+@pytest.fixture
+def arr_a():
+    return np.array([0, 1 / 3, np.nan, np.pi, np.nan])
```

Comment:
This array is used in multiple tests, so I moved to a fixture in the spirit of DRY.